### ISSUE TITLE
Beginning

### ISSUE THREAD

**Issue by nathanwells27** [nathanwells27/MentalHealthApp#1]
I am just beginning this development. I am not experienced and would appreciate any assistance with the development of this. Please let me know of any ideas of where to begin and any assistance you may have.

**Comment by luisfmelo** [comment]
Why not a web app?

**Comment by silverhoopedge** [comment]
If you want to start with WordPress and then create app from there look into https://apppresser.com/ (not free). Whatever the direction you select---HIPAA Compliance likely needed for saving/transmitting patient data.

**Comment by nathanwells27** [comment]
A webapp would most likely work, lol, thats why i needed input!

**Comment by nathanwells27** [comment]
@silverhoopedge, yes HIPAA will be the biggest for sure that needs to occur, thank you for the input

**Comment by nathanwells27** [comment]
if you guys are interested in helping further, i created an organization for this to further work with groups the link is here, https://github.com/WellsDirect/MentalHealthOrgApp
